README.md

```diff
@@ -148,6 +149,20 @@ For example, if your project doesn't use patch version numbers, you can set `ver
 
 Release Drafter requires full write, because GitHub does not offer a limited scope for only writing releases. **Don't install Release Drafter to your entire GitHub account — only add the repositories you want to draft releases on.**
 
+## Replacers
+
+With the `replacers` option you can provide string replaces for your template after all other template information is finished being processed.
+
+```yml
+replacers:
+  - regex: /JENKINS-(\d+)/g
+    replace: https://issues.jenkins-ci.org/browse/JENKINS-$1
+  - search: myname
```

Comment:
I think we made a big mistake by not quoting the string values in the configuration examples, and many people get tripped up once they use a semi-colon. Perhaps we should do this for these? It's easy for the `search` option, but I wonder if we need it for `regex` too?

I wonder what a better example for this one is too? 🤔

Reply:
I just changed the string quoting in #189